docs/release-testing.md

```diff
@@ -0,0 +1,107 @@
+## Generating and testing release bundles 
+
+More specific documentation for generating and testing bundles can be found [here](https://operator-framework.github.io/community-operators/testing-operators/).
+
+### Pre-requisites 
+Testing the bundle requires `opm`. To install opm, clone the operator-registry repository: 
+
+```
+git clone https://github.com/operator-framework/operator-registry
+```
+Next, cd into the directory and run `make build`. This will generate an `opm` executable in the `operator-registry/bin/` directory. Add that to your path, or substitue `opm` in the following commands with the full path to your `opm` executable. 
```

Comment:
```suggestion
Change into the directory and run `make build`. This will generate an `opm` executable in the `operator-registry/bin/` directory. Add the bin to your path, or substitute `opm` in the following commands with the full path to your `opm` executable.
```